rest_server/SiodbLoggerPool.go

```diff
@@ -0,0 +1,180 @@
+// Copyright (C) 2019-2020 Siodb GmbH. All rights reserved.
+// Use of this source code is governed by a license that can be found
+// in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"golang.org/x/sys/unix"
+)
+
+var (
+	LogFileSizeMin              uint64 = 1 * 1024 * 1024
+	LogFileExpirationTimeoutMin uint64 = 1 * 60
+)
+
+var (
+	FATAL_INIT_ERROR                   = 2
+	FATAL_UNABLE_TO_CLEANUP_TCP_BUFFER = 2
```

Comment:
why this at all matters? if you can't read remaining data from tcp connection, just close it. This should not be fatal error.